lib/mongo.ex

```diff
@@ -185,6 +185,21 @@ defmodule Mongo do
   end
 
   @doc """
+  Selects a single document in a collection and return the document or null
+  """
+  @spec findOne(Pool.t, collection, BSON.document, Keyword.t) :: BSON.document
+  def findOne(pool, coll, filter, opts \\ []) do
```

Comment:
@vans163 MongoDB seems to use `camelCase` for most of its functions- http://docs.mongodb.org/manual/reference/methods/ (and operators, afaik). I'd stick w/ `find_one` since that's what the rest of the driver uses.